tests/test_middlewares.py

```diff
@@ -55,9 +55,17 @@ def process_request(self, req, resp):
         req.path = req.path.replace('/base_path', '', 1)
 
 
+class AccessParamsMiddleware(object):
+
+    def process_resource(self, req, resp, resource, params):
```

Comment:
I think we should also test to ensure that a parameter that is part of the path shows up here in params.